sdk/remoterendering/mixed-reality-remote-rendering/package.json

```diff
@@ -88,7 +88,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^1.1.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
This is more complex than it seems. I recommend reverting this change. This project likely needs to re-do their recordings.